src/components/EditDeck/EditDeck.js

```diff
@@ -1,288 +1,289 @@
-import React, {Component} from 'react';
+import React, { Component } from "react";
```

Comment:
this is great use of prettier. I would also still recommend eslint also. It will help with some things that prettier doesn't.